src/routes/status.tsx

```diff
@@ -5,22 +5,287 @@
+      ),
+    },
+    {
+      title: 'Utxo Sum',
+      dataIndex: 'utxoSum',
+      key: 'utxoSum',
+      render: (value, record) => (
+        <TokenValue value={value} color={record.color} />
+      ),
+    },
+    {
+      title: 'Open Exit Sum',
+      dataIndex: 'exitSum',
+      key: 'exitSum',
+      render: (value, record) => (
+        <TokenValue value={value} color={record.color} />
+      ),
+    },
+    {
+      title: 'Status',
+      key: 'status',
+      render: (value, record: ColorDetails) => {
+        return record.plasmaBalance.gte(record.exitSum.add(record.utxoSum)) ? (
+          <Badge status="success" text="Ok" />
+        ) : (
+          <Badge status="warning" text="Invalid" />
+        );
+      },
+    },
+  ];
+
+  constructor(props) {
+    super(props);
+    reaction(() => this.props.exitHandler.address, this.getData, {
+      fireImmediately: true,
+    });
+  }
+
+  @observable exitDetails: ExitData[] = [];
+  @observable colorDetails: ColorDetails[] = [];
+
+  @action
+  private getData = async (): Promise<void> => {
+    const { exitHandler, explorer, web3, tokens, bridge } = this.props;
+    const exits = [];
+    if (!bridge.address)
+      await new Promise(resolve => reaction(() => bridge.address, resolve));
+    const fromBlock = await bridge.contract.methods.genesisBlockNumber().call();
+    const events = await exitHandler.contract.getPastEvents('ExitStarted', {
+      fromBlock,
+    });
+    await Promise.all(
+      events.map(async event => {
+        const utxoId = new Outpoint(
+          event.returnValues.txHash,
+          parseInt(event.returnValues.outIndex)
+        ).getUtxoId();
+        const exit = await exitHandler.contract.methods
+          .exits(web3.root.instance.utils.toHex(utxoId))
+          .call();
+        let suspect;
+        const exitTxHash = Tx.exit(
+          new Input(
+            new Outpoint(
+              event.returnValues.txHash,
+              parseInt(event.returnValues.outIndex)
+            )
+          )
+        ).hash();
+        const exitTx = await explorer.getTransaction(exitTxHash);
+        if (exitTx === undefined) {
+          suspect = true;
+        }
+        exits.push({ event, exit, suspect });
+      })
+    );
+
+    const uTxos = await web3.plasma.instance.getUnspentAll();
+    const addresses = await web3.plasma.instance.getColors();
+    const colors = await Promise.all(
+      addresses.map(async addr => {
+        return web3.plasma.instance.getColor(addr);
+      })
+    );
+    const colorDetails = await Promise.all(
+      colors.map(async color => {
+        const utxoSum = uTxos.reduce((acc, utXo) => {
+          return utXo.output.color === color
+            ? web3.root.instance.utils.toBN(utXo.output.value).add(acc)
+            : acc;
+        }, web3.root.instance.utils.toBN(0));
+
+        const exitSum = exits.reduce((acc, e) => {
+          return parseInt(e.exit.color) === color && !e.exit.finalized
```

Comment:
Don't forget to add second argument to `parseInt` or use `Number(e.exit.color)` instead